terraform/modules/cloudwatch/variables.tf

```diff
@@ -2,10 +2,14 @@ variable "stack_description" {
 
 }
 
-variable "sns_arn" {
+variable "sns_main_arn" {
```

Comment:
This looks great! Can we get variable types and descriptions throughout for the new variables we've added?